deploy-as-code/helm/charts/ifix-services/aggregator/values.yaml

```diff
@@ -0,0 +1,38 @@
+# Common Labels
+labels:
+  app: "aggregator"
+  group: "ifix"
+
+cron:
+  schedule: "0 0 * * *"  
+# Container Configs
+image:
+  repository: "aggregator"
+
+# Additional Container Envs
+env: |
+  - name: TOPIC_NAME
+    valueFrom:
+      configMapKeyRef:
+        name: reference-dashboard
+        key: aggregator-topic-name
+  - name: KAFKA_URL
+    valueFrom:
+      configMapKeyRef:
+        name: egov-config
+        key: kafka-brokers
+  - name: TIME_STRING
+    valueFrom:
+      configMapKeyRef:
+        name: reference-dashboard
+        key: aggregator-time-string
+  - name: DRUID_INSTANCE
+    valueFrom:
+      configMapKeyRef:
+        name: reference-dashboard
```

Comment:
Add the url for Druid [here](https://github.com/egovernments/iFix-DevOps/blob/9222bf4ed94f8a943c1c50dd7724b6ab5e3c69d9/deploy-as-code/helm/environments/ifix-dev.yaml#L61). 
It will be like
```druid-host: "http://druid-broker.olap:8082"```

And in the values.yaml file(this file). The `name` will be egov-config, and the `key` will be "druid-host".